testutils/src/main/java/com/microsoft/identity/internal/testutils/kusto/EstsKustoUtils.java

```diff
@@ -0,0 +1,193 @@
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.internal.testutils.kusto;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.microsoft.azure.kusto.data.ClientImpl;
+import com.microsoft.azure.kusto.data.ConnectionStringBuilder;
+import com.microsoft.azure.kusto.data.KustoOperationResult;
+import com.microsoft.azure.kusto.data.KustoResultSetTable;
+import com.microsoft.azure.kusto.data.exceptions.DataClientException;
+import com.microsoft.azure.kusto.data.exceptions.DataServiceException;
+import com.microsoft.azure.kusto.ingest.IngestClient;
+import com.microsoft.azure.kusto.ingest.IngestClientFactory;
+import com.microsoft.azure.kusto.ingest.IngestionMapping;
+import com.microsoft.azure.kusto.ingest.IngestionProperties;
+import com.microsoft.azure.kusto.ingest.exceptions.IngestionClientException;
+import com.microsoft.azure.kusto.ingest.exceptions.IngestionServiceException;
+import com.microsoft.azure.kusto.ingest.result.IngestionResult;
+import com.microsoft.azure.kusto.ingest.result.IngestionStatus;
+import com.microsoft.azure.kusto.ingest.result.OperationStatus;
+import com.microsoft.azure.kusto.ingest.source.FileSourceInfo;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.identity.internal.testutils.labutils.LabHelper;
+
+import java.net.URISyntaxException;
+import java.util.List;
+
+/**
+ * Utilities to interact with the Kusto Data SDK to perform operations on the ESTS Kusto Cluster.
+ */
+public class EstsKustoUtils {
+
+    private static final String TAG = EstsKustoUtils.class.getSimpleName();
+
+    private final static String ESTS_DATABASE_NAME = "ESTS";
+    private final static String ESTS_KUSTO_CLUSTER = "estswus2";
+    private final static String ESTS_KUSTO_APP_TENANT_ID = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
Should we consider loading this from a config so we don't need to recompile if changes?

Reply:
I guess same question for everything else; seems like there's lots of config we might consider loading dynamically

Reply:
Like a json config file that we store locally (in the repo)? or you mean like get these values from the LAB API?

Reply:
I was thinking something local

Reply:
Or, maybe, take all these attributes from a configuration of some sort and make an instance of this that way, rather than statics.  Whether code or config it's not that important.